src/main/kotlin/minesweeper/view/OutputView.kt

```diff
@@ -0,0 +1,13 @@
+package minesweeper.view
+
+import minesweeper.domain.GameBoard
+
+object OutputView {
+
+    fun printGameBoard(gameBoard: GameBoard) {
```

Comment:
꼭 필요한 객체만 받도록 하는게 어떨까요?
```suggestion
    fun printGameBoard(cells: Cells) {
```